main.tf

```diff
@@ -7,3 +7,21 @@ resource "azurerm_user_assigned_identity" "managed_identity" {
 
   tags = var.common_tags
 }
+
+locals {
+  mgmt_env = ( var.env == "sandbox") ? "cftsbox-intsvc" : "cftptl-intsvc"
+}
+
+resource "azurerm_user_assigned_identity" "mgmt_managed_identity" {
+
+  count = (var.env == "sandbox" || var.env == "prod" ) ? 1 : 0
```

Comment:
Do we want it only for sbox & prod ?

Reply:
i am just doing sbox ptl and prod ptl

Reply:
its already on all other envs. A bit hacky  as general pipeline doesn't create in those envs.